_data/members.yaml

```diff
@@ -0,0 +1,19 @@
+---
+# to get a gravatar for a user:
+# just run:
+# echo "http://www.gravatar.com/avatar/$(echo "rsg1egoman@gmail.com" | md5sum | sed 's/  -//')"
```

Comment:
I found the site we were looking for last night: https://en.gravatar.com/site/check/ 
Lets suggest people use that to find their gravatar link there. (the web version is preferable because `md5sum` is named `md5` on mac, and `who knows` on windows)